### PR TITLE
Feature/infrastructure

### PR DESCRIPTION
**Pull request by cableman**
First take on an k8s configuration for this part of the application.

### CODE REVIEW

File: infrastructure/docker/cover-service-faktor-export/etc/confd/templates/env.local.tmpl [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+APP_SECRET={{ getenv "APP_SECRET" "MySuperSecret" }}
+###< symfony/framework-bundle ###
+
+###> doctrine/mongodb-odm-bundle ###
+MONGODB_URL=mongodb://{{ getenv "APP_MONGODB_USER" "x" }}:{{ getenv "APP_MONGODB_PASSWORD" "y" }}@{{ getenv "APP_MONGODB_SERVER" "mongo" }}:{{ getenv "APP_MONGODB_PORT" "27017" }}
+MONGODB_DB={{ getenv "APP_MONGODB_DB" "db" }}
+###< doctrine/mongodb-odm-bundle ###
+
+###> redis ###
+REDIS_CACHE_DSN=redis://{{ getenv "APP_REDIS_CACHE_SERVER" "redis" }}:{{ getenv "APP_REDIS_CACHE_PORT" "6379" }}/{{ getenv "APP_REDIS_CACHE_DB" "10" }}
```
Comment (tuj) [inline infrastructure/docker/cover-service-faktor-export/etc/confd/templates/env.local.tmpl]: Default APP_REDIS_CACHE_DB should be 12

File: infrastructure/docker/nginx/etc/confd/templates/nginx.conf.tmpl [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+events {
+    worker_connections  2048;
+    multi_accept on;
+}
+
+
+http {
+    open_file_cache max=10000 inactive=5m;
+	open_file_cache_valid 5m;
```
Comment (tuj) [inline infrastructure/docker/nginx/etc/confd/templates/nginx.conf.tmpl]: Indention

File: infrastructure/k8s/app-deployment.yaml [hunk 3]
```diff
@@ -0,0 +1,166 @@
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  namespace: cover-service
+  name: cover-service-faktor-export-config
+data:
+  APP_REDIS_CACHE_SERVER: 'redis'
+  APP_REDIS_CACHE_PORT: '6379'
+  APP_REDIS_CACHE_DB: '10'
```
Comment (tuj) [inline infrastructure/k8s/app-deployment.yaml]: 12

File: infrastructure/k8s/readme.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+helm install faktor-mongo stable/mongodb --set mongodbUsername=faktor,mongodbPassword=faktor-1234,mongodbDatabase=faktor
+```
+
+Deploy the application.
+```bash
+kubectl apply -f app-deployment.yaml -f app-ingress.yaml -f app-secret.yaml
+```
+
+# Secret template
+Template to set secrets used be the application.
```
Comment (tuj) [inline infrastructure/k8s/readme.md]: used **by** the

**Review by tuj — CHANGES_REQUESTED** [review]

File: infrastructure/docker/cover-service-faktor-export/etc/confd/templates/env.local.tmpl (hunk 1 above)
Comment (cableman) [reply infrastructure/docker/cover-service-faktor-export/etc/confd/templates/env.local.tmpl]: Fixed

File: infrastructure/docker/nginx/etc/confd/templates/nginx.conf.tmpl (hunk 2 above)
Comment (cableman) [reply infrastructure/docker/nginx/etc/confd/templates/nginx.conf.tmpl]: fixed

File: infrastructure/k8s/app-deployment.yaml (hunk 3 above)
Comment (cableman) [reply infrastructure/k8s/app-deployment.yaml]: fixed

File: infrastructure/k8s/readme.md (hunk 4 above)
Comment (cableman) [reply infrastructure/k8s/readme.md]: fixed

**Review by tuj — APPROVED** [review]

**Pull request merged**